akka-http/src/main/scala/akka/http/scaladsl/server/Directives.scala

```diff
@@ -38,6 +38,7 @@ trait Directives extends RouteConcatenation
   with WebSocketDirectives
   with FramedEntityStreamingDirectives
   with AttributeDirectives
+  with CorsDirectives
```

Comment:
This will cause problems for users having the original cors module on classpath because names will collide. Maybe that is fine though and the recommendation is to move over to the built in cors support?